src/mediaelement.js

```diff
@@ -86,6 +86,16 @@ WaveSurfer.util.extend(WaveSurfer.MediaElement, {
         this.playbackRate = value || 1;
         this.media.playbackRate = this.playbackRate;
     },
+    
+    /**
+     * Get the audio source playback rate.
+     */
+    getPlaybackRate: function () {
```

Comment:
Actually, there's no need for this method, as it's inherited from the Web Audio backend.